.github/CODEOWNERS

```diff
@@ -102,6 +112,8 @@
 # PRLabel: %Content Safety
```

Comment:
I don't see another entry for this one.  We'll want a `ServiceLabel` element here so these folks get assigned via "Service Attention".